insuree/models.py

```diff
@@ -310,6 +310,27 @@ class InsureePolicy(core_models.VersionedModel):
     audit_user_id = models.IntegerField(db_column='AuditUserID')
     # row_id = models.BinaryField(db_column='RowID', blank=True, null=True)
 
+    @classmethod
+    def filter_queryset(cls, queryset=None):
+        if queryset is None:
+            queryset = cls.objects.all()
+        return queryset
+
+    @classmethod
+    def get_queryset(cls, queryset, user):
+        queryset = cls.filter_queryset(queryset)
+        # GraphQL calls with an info object while Rest calls with the user itself
+        if isinstance(user, ResolveInfo):
+            user = user.context.user
+        if settings.ROW_SECURITY and user.is_anonymous:
+            return queryset.filter(id=-1)
+        if settings.ROW_SECURITY:
+            dist = UserDistrict.get_user_districts(user._u)
+            return queryset.filter(
+                location__parent__parent_id__in=[l.location_id for l in dist]
```

Comment:
we might have an issue here because insuree policy has not Parent, you must go through insuree_familly_location_parent_parent_id or insuree_location_parent_parent_id but the later will be mostly empty

You did it well in schema line 206

Reply:
Ok @delcroip . I changed this. You can re-check this. Thx!